Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -375,6 +398,21 @@ void Close()
 
 	SAFE_RELEASE(context);
 	ULONG references = device->Release();
+
+#if defined(_DEBUG) || defined(DEBUGFAST)
+	if (debug)
+	{
+		--references; // we didn't release the debug interface yet
```

Comment:
the debug interface is released at 412. The debug interface itself increases the refcount of the device, so i have to substract that.

I have to do the debug check after i release the device, because otherwise I don't know the refCount of the device. Also, debug->ReportLiveDeviceObjects(D3D11_RLDO_SUMMARY | D3D11_RLDO_DETAIL); would otherwise show another live reference.